### PR TITLE
Add Coordinator Module and Service to housing

### PR DESCRIPTION
**Pull request by gpyofavv**
# housingxyz :: pull-request

## reviewer

- @escalonn

this project is maintained under the terms of the [Contribution Guidelines][contribution-guidelines-url].

[contribution-guidelines-url]: https://github.com/housingxyz/housingxyz/blob/master/.github/CONTRIBUTING.md 'CONTRIBUTION GUIDELINES'

### CODE REVIEW

**Comment by jacobdavis1** [comment]
Builds are failing though? Fix those first if you can.

**Comment by gpyofavv** [comment]
Mistake occured: Branch didn't pull some changes to master earlier on. Changed in latest commit, will build.

**Comment by gpyofavv** [comment]
Builds successfully

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            }
+        },
+        {
+            "name": ".NET Core Attach",
+            "type": "coreclr",
+            "request": "attach",
+            "processId": "${command:pickProcess}"
+        }
+    ]
+}
```
Comment (escalonn) [inline .vscode/launch.json]: remove these `.vscode` files from the top level dir, don't re-add them

**Comment by escalonn** [comment]
merge latest from master, then hopefully it will pass

**Pull request merged**